File: src/test/java/javabot/JavabotConfigTest.java
package javabot;

import com.google.inject.Inject;
import org.testng.Assert;
import org.testng.annotations.Test;

public class JavabotConfigTest extends BaseTest {

    @Inject
    private JavabotConfig javabotConfig;

    @Test
    public void testConfig() {
        Assert.assertNotEquals(javabotConfig.nick(), "javabot");
        Assert.assertNotEquals(javabotConfig.databaseName(), "javabot");
    }
}

File: src/test/kotlin/javabot/JavabotConfigTest.kt
package javabot

import io.quarkus.test.junit.QuarkusTest
import org.junit.jupiter.api.Assertions.assertNotEquals
import org.junit.jupiter.api.Test

@QuarkusTest
class JavabotConfigTest : BaseTest() {

    private val javabotConfig: JavabotConfig by lazy {
        injector.getInstance(JavabotConfig::class.java)
    }

    @Test
    fun testConfig() {
        assertNotEquals("javabot", javabotConfig.nick())
        assertNotEquals("javabot", javabotConfig.databaseName())
    }
}
